bundle/manifests/oadp-operator.clusterserviceversion.yaml

```diff
@@ -317,16 +308,10 @@ spec:
         displayName: StartTimestamp
         path: startTimestamp
       version: v2alpha1
-    - description: DataProtectionApplication represents configuration to install a
-        data protection application to safely backup and restore, perform disaster
-        recovery and migrate Kubernetes cluster resources and persistent volumes.
-      displayName: DataProtectionApplication
+    - description: DataProtectionApplication is the Schema for the dpa API
+      displayName: Data Protection Application
       kind: DataProtectionApplication
       name: dataprotectionapplications.oadp.openshift.io
-      statusDescriptors:
-      - description: Conditions defines the observed state of DataProtectionApplication
-        displayName: Conditions
-        path: conditions
```

Comment:
Where possible, please add this back programmatically using operatorsdk's API Markers

read: https://sdk.operatorframework.io/docs/building-operators/golang/references/markers/
https://github.com/openshift/console/blob/master/frontend/packages/operator-lifecycle-manager/src/components/descriptors/reference/reference.md

Reply:
done :smile: